File: PYTHON/Program/OutputDevice.py
from Komponen import Komponen

class OutputDevice(Komponen):
    def __init__(self, jenisOutput="", outputLatency="", merk="", nama=""):
        super().__init__(merk, nama)
        self.jenisOutput = jenisOutput
        self.outputLatency = outputLatency

    def setJenisOutput(self, jenisOutput):
        self.jenisOutput = jenisOutput

    def setOutputLatency(self, outputLatency):
        self.outputLatency = outputLatency

    def getJenisOutput(self):
        return self.jenisOutput

    def getOutputLatency(self):
        return self.outputLatency

File: JAVA/Program/OutputDevice.java
package Program;
public class OutputDevice extends Komponen {
    private String jenisOutput;
    private String outputLatency;

    public OutputDevice() {
    }

    public OutputDevice(String jenisOutput, String outputLatency, String merk, String nama) {
        super(merk, nama);
        this.jenisOutput = jenisOutput;
        this.outputLatency = outputLatency;
    }

    public void setJenisOutput(String jenisOutput) {
        this.jenisOutput = jenisOutput;
    }

    public void setOutputLatency(String outputLatency) {
        this.outputLatency = outputLatency;
    }

    public String getJenisOutput() {
        return jenisOutput;
    }

    public String getOutputLatency() {
        return outputLatency;
    }
}
